Public/Src/Cache/ContentStore/Library/BuildXL.Cache.ContentStore.Library.dsc

```diff
@@ -13,11 +13,18 @@ namespace Library {
             ...(BuildXLSdk.isDotNetCoreBuild ? [
                 // TODO: This is to get a .Net Core build, but it may not pass tests
                 importFrom("System.Data.SQLite.Core").withQualifier({targetFramework: "net461"}).pkg,
+                
+                importFrom("Microsoft.Azure.Kusto.Data.NETStandard").pkg,
+                importFrom("Microsoft.Azure.Kusto.Ingest.NETStandard").pkg,
+                importFrom("Microsoft.Azure.Kusto.Cloud.Platform.Azure.NETStandard").pkg,
+                importFrom("Microsoft.Azure.Kusto.Cloud.Platform.NETStandard").pkg,
+                importFrom("Microsoft.Extensions.PlatformAbstractions").withQualifier({targetFramework: "net472"}).pkg,
             ] :
             [
                 importFrom("System.Data.SQLite.Core").pkg,
                 NetFx.System.Data.dll,
                 NetFx.System.Runtime.Serialization.dll,
+                importFrom("Microsoft.Azure.Kusto.Ingest").withQualifier({targetFramework: "net462"}).pkg,

```

Comment:
>importFrom("Microsoft.Azure.Kusto.Ingest").withQualifier({targetFramework: "net462"}).pkg, [](start = 16, length = 90)

Its a bit weird for ContentStore.Library to have a dependency on Kusto. Can the Kusto logic be moved to App assembly?

Reply:
I can do that.

Reply:
moved